.gitignore

```diff
@@ -47,5 +49,6 @@ lib*.so.*
 /test/thrash_threads[1-6]
 /test/*.tmp
 /test/*.tmp.*
+/test/*.new
```

Comment:
`*.new` files are output files representing failed test cases and are not cleaned by `make clean`.  They need to be visible in `git status` so that you can see what tests have failed and can see them to remove them by hand.  Hence they are intentionally not in _.gitignore_.  (If that policy were to change, further changes would be necessary — perhaps just adding them to `make clean` too.)